### ISSUE TITLE
daily or weekly or monthly

### ISSUE THREAD

**Issue by alisonasanchez** [AdobeDocs/experience-manager-cloud-service.en#65]
Issue in ./help/operations/maintenance.md

* The doc says "/apps/settings/granite/operations/maintenance/granite*weekly or granite*daily". This leads me to believe that only daily and weekly purges are possible. What about monthly? Are those excluded for a reason? 
* It also seems that it is not possible to have a purge twice  a week or twice a month. Only once per time frame. Can this be confirmed and documented? 
* Also, there should be an explanation of the code samples. Would also be nice to know what the default values are.

**Comment by alisonasanchez** [comment]
Also, there should be an explanation of the code samples.

**Comment by alisonasanchez** [comment]
Would also be nice to know what the default values are.

**Comment by aheim0** [comment]
Thanks for raising this - we will investigate.

**Comment by aheim0** [comment]
@adobe export issue to Jira project CQDOC

**Comment by raimanS** [comment]
The page has been updated to cover the first two points. For the last point, the default values can be checked in the OSGI console. The code samples will be reviewed/replaced at a later date and will be addressed in a separate issue. Thanks.
